common/lib/xmodule/xmodule/seq_module.py

```diff
@@ -455,6 +477,11 @@ def _render_student_view_for_items(self, context, display_items, fragment):
                 'path': " > ".join(display_names + [item.display_name_with_default]),
             }
 
+            completion_waffle = self.runtime.service(self, 'completion_waffle')
+            if completion_waffle.visual_progress_enabled(self.runtime.course_id):
```

Comment:
So we want a separate waffle mechanism for the new course outline, but not for this feature?

Reply:
Also, gentle reminder that this will return `True` as soon as we deploy, because the switch is already on in prod.

Reply:
@iloveagent57 Yes. Course outline is different because it currently do not leverage completion data like this one does. It's largely an UX experience update.
I checked with Sham and it is OK for this to be visible to all learners once deployed.